plugins/CoreHome/angularjs/reporting-menu/reportingmenu.controller.js

```diff
@@ -235,6 +262,12 @@
 
             $('#loadingError').hide();
 
+            if ($scope.currentSubcategory != 'Live_VisitorLog') {
```

Comment:
also the real time widget page etc. Eventually would maybe need to let plugins hook into this